assets/src/conversion/index.js

```diff
@@ -104,130 +114,115 @@ class Conversion extends Component {
-						) }
-					</p>
-					<input
-						type="submit"
-						className="large"
-						value={ __( 'Reset Conversion' ) }
-						onClick={ this.handleOnClickResetConversion }
-					/>
-				</div>
-			);
-		} else {
-			return (
-				<div className="ncc-page">
-					<h1>{ __( 'Run Conversion' ) }</h1>
-
-					<p>
-						<b>
-							<u>{ __( 'Note carefully' ) }</u>
-						</b>
-						{ ' -- ' }
-						{ __(
-							'once started, the conversion should not be interrupted! Your browser page needs to remain active until conversion is complete. You may, however, manually open multiple browser tabs once the conversion starts, and each of them will fetch and convert another batch of posts in parallel. Running multiple tabs speeds up the conversion. Recommended max number of tabs is 4.'
-						) }
-					</p>
-
-					<h3>{ __( 'Planned Conversion' ) }</h3>
-					<ul>
-						<li>{ __( 'the entire queued content will be converted to Gutenberg Blocks' ) }</li>
-						<li>{ __( 'this page will automatically reload for every batch' ) }</li>
-						<li>
-							{ __( 'number of posts/entries to be converted:' ) }{' '}
-							<b>
-								<u>{ queuedEntries }</u>
-							</b>
-						</li>
-						<li>
-							{ __( 'total conversion batches queued' ) }:{' '}
-							<b>
-								<u>{ maxBatch }</u>
-							</b>
-						</li>
-						<li>
-							{ __( 'batch size (posts per batch):' ) }{' '}
-							<b>
-								<u>{ conversionBatchSize }</u>
-							</b>
-						</li>
-					</ul>
-
-					<br />
-
-					<input
-						type="submit"
-						className="large"
-						value={ __( 'Start Conversion' ) }
-						onClick={ this.handleOnClickInitializeConversion }
-					/>
-
-					<br />
-					<br />
-
-					{ !! someConversionsFailed && (
-						<div>
-							<h3>{ __( 'Retry Converting Failed Posts' ) }</h3>
+				<Fragment>
+					<div className="newspack-logo-wrapper">
+						<NewspackLogo />
+					</div>
+					<Grid>
+						<FormattedHeader
+							headerIcon={ <UnarchiveIcon /> }
+							headerText={ __( 'Run conversion' ) }
+							subHeaderText={ __( 'A converson is currently in progress.' ) }
+						/>
+						<Card>
+							<Notice noticeText={ __( 'A designated browser tab has already started to convert your content.' ) } isPrimary />
+							<hr />
+							<h2>{ __( 'Conversion progress' ) }</h2>
+							<TextControl
+								label={ __( 'Conversion batches processed' ) }
+								disabled={ true }
+								value={ queuedBatchesCsv }
+							/>
+							<TextControl
+								label={ __( 'Total conversion batches queued' ) }
+								disabled={ true }
+								value={ maxBatch }
+							/>
+							<TextControl
+								label={ __( 'Conversion batch size (entries per batch)' ) }
+								disabled={ true }
+								value={ conversionBatchSize }
+							/>
+							<hr />
+							<h2>{ __( 'Reset ongoing conversion' ) }</h2>
```

Comment:
Would it be OK if we moved this block above the ["Conversion progress" block](https://github.com/Automattic/newspack-content-converter/pull/41/files#diff-f72953981e26d1c79101da50fe6734f0R130)? Looking at it here, since "Reset" is the more important part, how about we push it upwards to be visible first?

![image](https://user-images.githubusercontent.com/29167323/72438054-58ab3180-37a4-11ea-8a7f-cf9760a32bdd.png)